qt/widgets/instrumentview/src/InstrumentDisplay.cpp

```diff
@@ -72,4 +72,19 @@ void InstrumentDisplay::updateView(bool picking) {
     m_qtDisplay->updateView(picking);
   }
 }
+
+/// Return the size of the OpenGL or Qt display widget in logical pixels
+QSize InstrumentDisplay::widgetDimensions() const {
```

Comment:
I found that I was able to remove this implementation when I switched to the OpenGLWidget. Is this just now able to return `m_instrumentDisplay->currentWidget()->size()` too?

Reply:
I don't think this function is getting called anywhere so I'll just remove.

I had adjusted for the fact you'd removed glWidgetDimensions from the InstrumentWidget class when I rebased this PR onto main (see new L612 in InstrumentWidget.cpp for example in this PR where I call `m_instrumentDisplay->currentWidget()->size()`). However it looks like I'd moved glWidgetDimensions (with a rename) into the InstrumentDisplay class and this bit of code got left behind.